JAGPropertyConverterTests/JAGPropertyConverterTest.m

```diff
@@ -72,9 +72,19 @@ - (void) testToDictionaryJSON {
     NSDictionary *dict = [converter convertToDictionary:model];
     NSLog(@"Converted to dictionary.");
     [self assert:model isEqualTo:dict];
+
+    STAssertEquals(dict[@"keypathProperty1"], model.modelProperty.testModelID, @"JSON Dictionary should have set the keypath property properly.");
+    STAssertEquals(dict[@"keypathProperty2"], model.modelProperty.modelProperty.testModelID, @"JSON Dictionary should have set the keypath property properly.");
+}
+
+- (void) testKeypathPropertiesToDictionaryJSON {
```

Comment:
you have swapped this method and the above method. this one is about Keypath, but you are only testing for normal property in dict. 

above method is testing normal dict, but you are testing for keypaths.